apps/web/components/FrontpageTabs/illustrations/ManOnBenchIllustration.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+
+function ManOnBenchIllustration() {
```

Comment:
other projects need these illustrations. I suggest that these illustrations are moved to a shared library.

Reply:
Good call